app/lib/world/world.coffee

```diff
@@ -693,7 +692,7 @@ module.exports = class World
 
   clampHeroHealth: (level) ->
     hero = _.find @thangs, id: 'Hero Placeholder'
-    if hero?
+    if hero? and level.clampHeroHealth?
```

Comment:
Why are we saving this info on the level, instead of call `me.tryClampHeroHealth()` directly?

Reply:
You probably don't want an `?` in this check, since we seem to be setting to true or not.  This will accept a value of false as well.

Reply:
It's probably faster to `return unless level.clampheroHealth` before the `hero =` line.